package-lock.json

```diff
@@ -392,14 +392,14 @@
       "dev": true
     },
     "asn1.js": {
-      "version": "4.9.2",
```

Comment:
this seems like  it might be a lot of changes for just adding a couple of dependencies, what version of npm are you using out of curiosity?